smalldata_tools/SmallDataUtils.py

```diff
@@ -159,6 +159,7 @@ def mfxDetectors(beamCodes=[[162, 120],[]]):
     dets.append(feeBldDetector('FEE-SPEC0','feeBld'))
     dets.append(aiDetector('MFX-AIN-01','ai')) 
     dets.append(ttDetector(baseName='MFX:TTSPEC:'))
+    dets.append(ttDetector(baseName='MFX:TIMETOOL:'))
```

Comment:
Only one should be there (unless you have data where the old style name is actually used....)

Reply:
So the comment can go.

Reply:
I did not have the chance to test it properly with the timetool in the DAQ. This should indeed be fixed

Reply:
TTSPEC was the old name is whatever PV you are hosting in the notepad. We are used TIMETOOL now, so I don't think MFX has any data where it's called TTSPEC. XPP for sure does and likely CXI as well, so there we add both to make sure we find the relevant data for both old&new data. So removing the TTSPEC line is what should happen with 99% certainty.